src/graphql/prescriptions/queries.tsx

```
@@ -147,10 +147,9 @@ export const ANALYSIS_ENTITY_QUERY = (requestId: string) => gql`
         text
       }
       code @flatten {
-        coding(system: "http://fhir.cqgc.ferlab.bio/CodeSystem/analysis-request-code")
-          @flatten
-          @first {
+        coding{
```

Revert that change (also it had side-effects making prescription.code string -> array making `src/views/Prescriptions/Detail/AnalysisCard/index.tsx:48` silently fails, once revert will be fine)

Easy change to retreive the data you want is in that file but different place:

**ANALYSIS_PATIENT_FRAGMENT**

Update the **requests** part as bellow;

```
 requests: ServiceRequestList(_reference: patient, based_on: "${requestId}") {
      id
      authoredOn
      code @flatten {
        coding{
          code
          system
        }
      }
...
```